frontend/src/app/homepage-offer/offers.service.ts

```diff
@@ -41,4 +48,24 @@ export class OffersService {
     return `${environment.djangoRoot}/offers/${offer.slug}/${offer.id}/join`;
   }
 
+  postOffer(offer: Offer) {
+    console.log(offer)
+    return this.http.post(`${environment.apiRoot}/offers/`, offer, { withCredentials: true })
+    .map(response => {
+      if (response.status === 201) {
+        return 'success';
```

Comment:
I am not sure if we sould return a string here. @magul Shouldn't the newly created offer be returned from the backend?